<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>801</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Image Control</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSlider" name="brightnessS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Luminosit√©</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Convolution</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ConvolutionChoice">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Identity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edge Detection 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edge Detection 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edge Detection 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sharpen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Box Blur 3x3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Box Blur 5x5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Box Blur 7x7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian Blur 3x3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian Blur 5x5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unsharp Masking 5x5</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="lumB">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>20</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>225</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Redimensionner</string>
     </property>
    </widget>
    <widget class="QSlider" name="resizeS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="resizeB">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="angleT">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateB">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="dilationB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Dilatation </string>
     </property>
    </widget>
    <widget class="QPushButton" name="erosionB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Erosion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openingB">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>510</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ouverture </string>
     </property>
    </widget>
    <widget class="QPushButton" name="closingB">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Fermeture </string>
     </property>
    </widget>
    <widget class="QSlider" name="morphTrans">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Transformations Morphologiques </string>
     </property>
    </widget>
    <widget class="QPushButton" name="binaryB">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>430</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Binaire </string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="binaryB2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>590</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Binaire</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D√©tection de Contour </string>
     </property>
    </widget>
    <widget class="QPushButton" name="ContourD_B">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>590</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QSlider" name="ContourD_V">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>640</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>680</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>710</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>D√©tection des Caract√®res </string>
     </property>
    </widget>
    <widget class="QPushButton" name="C_Detection">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>710</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addimageB">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>721</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ouvrir une Image </string>
    </property>
   </widget>
   <widget class="QLabel" name="photoL">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>841</width>
      <height>771</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="save_img">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
